packages/jmix-react-ui/src/ui/notifications/Notifications.consts.ts

```diff
@@ -0,0 +1,7 @@
+export enum NotificationTypes {
+  SUCCES = 'success',
+  ERROR = 'error',
+  INFO = 'info',
+  WARNING = 'warning',
+  WARN = 'warn'
```

Comment:
I think those are just aliases in Antd, so we shouldn't have them both in our API.